lib/kafka_ex.ex

```diff
@@ -519,6 +519,24 @@ defmodule KafkaEx do
 
   end
 
+  @doc """
+  Check to see if worker (server) is ready and connected
+
+  Optional arguments(KeywordList)
+  - worker_name: the worker we want to run this metadata request through, when none is provided the default worker `:kafka_ex` is used
+
+  ## Example
```

Comment:
I would add in an example of passing in the optional arguments as well, maybe consider adding a `@spec` to describe that it is a `KeywordList` without having to check the `@doc`

Reply:
- [ ] add example of passing in the optional arguments

typespec should exist for the Keyword list to capture all other uses, but it does not, so I would consider that a good change to make outside the scope of this MR.

Reply:
:+1:

Reply:
done (how do we close a GitHub comment?!)